templates/android/library/src/main/java/io/package/services/Service.kt.twig

```diff
@@ -64,7 +64,11 @@ class {{ service.name | caseUcfirst }}(client: Client) : Service(client) {
 
         val apiParams = mutableMapOf<String, Any?>(
         {%~ for parameter in method.parameters.query | merge(method.parameters.body) %}
-            "{{ parameter.name }}" to {{ parameter.name | caseCamel }},
+            {%~ if method.name | caseLower == "createexecution" and parameter.name == 'body' %}
```

Comment:
Why are we checking the method name? (Also typo in method name). We should avoid matching method names